test_integration_app/main.py

```diff
@@ -0,0 +1,93 @@
+    assert (objUtils.isOK(objEvents.chargeback()) == True)
+    assert (objUtils.isOK(objEvents.content_status()) == True)
+    assert (objUtils.isOK(objEvents.create_account()) == True)
+    assert (objUtils.isOK(objEvents.create_content_comment()) == True)
+    assert (objUtils.isOK(objEvents.create_content_listing()) == True)
+    assert (objUtils.isOK(objEvents.create_content_message()) == True)
+    assert (objUtils.isOK(objEvents.create_content_post()) == True)
+    assert (objUtils.isOK(objEvents.create_content_profile()) == True)
+    assert (objUtils.isOK(objEvents.create_content_review()) == True)
+    assert (objUtils.isOK(objEvents.create_order()) == True)
+    assert (objUtils.isOK(objEvents.flag_content()) == True)
+    assert (objUtils.isOK(objEvents.link_session_to_user()) == True)
+    assert (objUtils.isOK(objEvents.login()) == True)
+    assert (objUtils.isOK(objEvents.logout()) == True)
+    assert (objUtils.isOK(objEvents.order_status()) == True)
+    assert (objUtils.isOK(objEvents.remove_item_from_cart()) == True)
+    assert (objUtils.isOK(objEvents.security_notification()) == True)
+    assert (objUtils.isOK(objEvents.transaction()) == True)
+    assert (objUtils.isOK(objEvents.update_account()) == True)
+    assert (objUtils.isOK(objEvents.update_content_comment()) == True)
+    assert (objUtils.isOK(objEvents.update_content_listing()) == True)
+    assert (objUtils.isOK(objEvents.update_content_message()) == True)
+    assert (objUtils.isOK(objEvents.update_content_post()) == True)
+    assert (objUtils.isOK(objEvents.update_content_profile()) == True)
+    assert (objUtils.isOK(objEvents.update_content_review()) == True)
+    assert (objUtils.isOK(objEvents.update_order()) == True)
+    assert (objUtils.isOK(objEvents.update_password()) == True)
+    assert (objUtils.isOK(objEvents.verification()) == True)
+        
+    # Decision APIs
+
+    assert (objUtils.isOK(ObjDecision.apply_user_decision()) == False)
+    assert (objUtils.isOK(ObjDecision.apply_order_decision()) == False)
+    assert (objUtils.isOK(ObjDecision.apply_session_decision()) == False)
+    assert (objUtils.isOK(ObjDecision.apply_content_decision()) == False)
+    assert (objUtils.isOK(ObjDecision.get_user_decisions()) == True)
+    assert (objUtils.isOK(ObjDecision.get_order_decisions()) == True)
+    assert (objUtils.isOK(ObjDecision.get_content_decisions()) == True)
+    assert (objUtils.isOK(ObjDecision.get_session_decisions()) == True)
+    assert (objUtils.isOK(ObjDecision.get_decisions()) == True)
+
+    # Workflows APIs
+
+    assert (objUtils.isOK(objWorkflow.synchronous_workflows()) == True)
+
+    # Score APIs
+
+    assert (objUtils.isOK(objScore.get_user_score()) == False)
+
+    # Verification APIs
+
+    assert (objUtils.isOK(objVerification.send()) == True)
+    assert (objUtils.isOK(objVerification.resend()) == True)
+    assert (objUtils.isOK(objVerification.check()) == True)
```

Comment:
@haneeshv how this one gonna work?

Reply:
suggested:
`res = objVerification.check()
assert (objUtils.isOK(res) == True)
assert. res.status == 50
`